influxql.gen.go.tmpl

```diff
@@ -42,6 +43,10 @@ func (a *{{.name}}PointAggregator) AggregateBatch(b *models.Batch) {
 			// We need to populate the Aux fields
 			{{.name}}PopulateAuxFieldsAndTags(ap, a.topBottomInfo.FieldsAndTags, p.Fields, p.Tags)
 		}
+
+    if a.isSimpleSelector {
```

Comment:
Can you convert the spaces to tabs in these tmpl files since they are supposed to read as go files? Thanks